main.go

```diff
@@ -39,29 +45,33 @@ func main() {
 			Destination: &verbose,
 			Hidden:      false},
 	}
+	app.Before = func(*cli.Context) error {
+		rpcUrl = getRPCURL(network, rpcUrl, testnet)
```

Comment:
I went back and forth on whether reusing the `rpcUrl` variable was confusing or not.